server/venueless/core/permissions.py

```diff
@@ -7,7 +7,9 @@ class Permission(Enum):
     WORLD_ANNOUNCE = "world:announce"
     WORLD_SECRETS = "world:secrets"
     WORLD_API = "world:api"
-    WORLD_ROOMS_CREATE = "world:rooms.create"
+    WORLD_ROOMS_CREATE_STAGE = "world:rooms.create.stage"
```

Comment:
+0 on dropping the leading "`WORLD_`" here, which doesn't really provide any extra info and kinda sounds weird.

Reply:
It does! It's a permission that is checked on world level not on room level, so your room-level permissions are irrelevant. That's kinda obvious since the room does not exists but starting with ROOMS would be pretty inconsistent with the actual functionality.